languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -35303,7 +35303,18 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TOURS_CRITIQUES" name="Tours critiqués divers (barbarismes, impropriétés, solécismes, etc.)">
-    <rulegroup id="POUR_PAS_QUE" name="familier : pour (ne) pas que...">
+      <rule id= "ALTERNATIVE_OU_POSSIBILITE" name="alternative ou possibilité" default="temp_off">
+          <pattern>
+              <token regexp="yes">autre|deux</token>
+              <marker><token regexp="yes">alternative(s)?</token></marker>
+          </pattern>
+          <message>Une alternative est un choix entre deux possibilités.</message>
```

Comment:
I'd like you to change this message, it feels to teacher-like. Something along the lines of "Cette expression est familière. Dans un contexte formel, il est préférable d'écrire [...] car une alternative est un choix entre deux possibilités."

Qui plus est : Si quelqu'un me propose une alternative à quelque chose, ne pourrais-je pas demander "N'y a-t-il pas d'autres alternatives ?", c'est-à-dire, n'y a-t-il pas une troisième ou quatrième possibilité ? En soi, le mot "alternative" n'a rien de dual, je crois...